### PR TITLE
Added my project to the project folder.

### PR DESCRIPTION
**Pull request by akanksha-raghav**

### CODE REVIEW

File: projects/akanksha-raghav/Minor_Project.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  - JavaScript
+  - CSS
+  - HTML / CSS
+  - React
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - ReactJs Application
+  - Begineer friendly
+  - Education
+  - Javascript Application
```
Comment (deammer) [inline projects/akanksha-raghav/Minor_Project.mdx:15]: **suggestion:** I recommend shortening the list of tags to keep the site optimized.
```suggestion
  - Education
```

**Review by deammer — COMMENTED** [review]
Hi @akanksha-raghav, welcome to OSS Port! We're excited to feature your project on our site.

I want to apologize because our instructions were unclear: there is no need to upload your entire project, all you need to do is create a `.mdx` file. We will improve our onboarding instructions to make the process clearer. In the meantime, please delete all the files you created except for the `Minor_Project.mdx` file.

Additionally, I've recommended a change to the tags you provided. We're trying to keep the list of tags short according to our [tag policy](https://github.com/codesee-io/oss-port#tag-policy).

Let me know if you have any questions!

File: projects/akanksha-raghav/Minor_Project.mdx (hunk 1 above)
Comment (akanksha-raghav) [reply projects/akanksha-raghav/Minor_Project.mdx:15]: okay i will be doing the necessary changes

**Comment by akanksha-raghav** [comment]
I have done the necessary changes kindly check the pr @deammer

**Comment by jleven** [comment]
Thanks so much @akanksha-raghav!  I think there's just one more detail. When I follow the url to your CodeSee Map, I'm not able to see a map!

Can you make sure it's the right url? Double check it's the right map, and please be sure to click the "Share" button in the upper-right, choose "public", and copy the url from there.

Thanks so much!

**Comment by akanksha-raghav** [comment]
> Thanks so much @akanksha-raghav! I think there's just one more detail. When I follow the url to your CodeSee Map, I'm not able to see a map!
> 
> Can you make sure it's the right url? Double check it's the right map, and please be sure to click the "Share" button in the upper-right, choose "public", and copy the url from there.
> 
> Thanks so much!

I have made it public. Kindly check @jleven

**Review by jleven — APPROVED** [review]
Everything looks great! Thanks so much for bearing with us.

You were one of the very first maintainers to add a project, and so you ran into a number of things that weren't as smooth as we would like. But, thanks to your help, we have smoothed those out for future contributors 🙏

**Comment by akanksha-raghav** [comment]
> Everything looks great! Thanks so much for bearing with us.
> 
> You were one of the very first maintainers to add a project, and so you ran into a number of things that weren't as smooth as we would like. But, thanks to your help, we have smoothed those out for future contributors

Thank you!

**Pull request merged**